src/main/java/org/example/daos/JobRoleDao.java

```diff
@@ -12,30 +13,50 @@
 import java.util.List;
 
 public class JobRoleDao {
-    public List<JobRole> getJobRoles() throws SQLException {
-        List<JobRole> jobRoles = new ArrayList<>();
+
+    public List<JobRoleResponse> getJobRoles(
+            final int offset, final int limit) throws SQLException {
+        List<JobRoleResponse> jobRoles = new ArrayList<>();
         try (Connection connection = DatabaseConnector.getConnection()) {
             String query = "SELECT jobRoleId, roleName, location,"
                     + " Capability.capabilityName, Band.bandName, "
                     + "closingDate FROM `job-roles`"
                     + " JOIN Capability using(capabilityId)"
                     + " JOIN Band using(bandId)"
-                    + " WHERE statusId = 1;";
+                    + " WHERE statusId = 1"
```

Comment:
Replace this magic number with a meaningful variable. '1' gives no context of what we are doing

Reply:
Fixed all Magic number in the class